pipeline.jsonc

```diff
@@ -4,7 +4,7 @@
         "environment":
         {
             "HELPER_ENABLED": true,
-            "IMAGE_TAG": "1.9.0-develop"
+            "IMAGE_TAG_VERSION": "v1"
```

Comment:
Blockchain has asked us to use the `pipeline.jsonc` file only when we need to override default pipeline behavior, but these variables are intended to create default pipeline behavior and should be implemented in one of the following places:
- Pipeline upload script
- Pipeline YAML
- Pipeline terraform

`IMAGE_TAG_VERSION` should be deleted here and implemented in `auto-buildkite-pipelines` unless it is overriding some default behavior.